spec/models/user_spec.rb

```diff
@@ -1,5 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'tutor' do
+    before(:each) do
+      @user = User.new
```

Comment:
So a big problem we have at work is creating unnecessary objects. It didn't matter in the beginning but it became a big problem as time went on. So instead of `before_each` (or in this case `before(:each)`), if the object isn't modified, it would just be done one (using `before`).